requirements.txt

```diff
@@ -8,3 +8,4 @@ requests==2.20.0
 retry==0.9.2
 tqdm==4.46.0
 waiting==1.4.1
+dnspython==1.16.0
```

Comment:
we will have lint validation and it verifies that packages in requirements are in alphabetical order

Reply:
done